### PR TITLE
Add markerNames property

### PR DESCRIPTION
**Pull request by dpt**
This adds a new property `markerNames` to Animation which returns an array of all marker names.

There's a couple of C&P fixes in there too.

### CODE REVIEW

**Comment by dpt** [comment]
#849 is related.

File: lottie-swift/src/Private/Model/Animation.swift [hunk 1]
```diff
@@ -57,6 +57,13 @@ public class Animation: Codable {
   let markers: [Marker]?
   let markerMap: [String : Marker]?
   
+  /// Return all marker names, in order, or an empty list if none are specified
+  public var allMarkerNames: [String]
```
Comment (buba447) [inline lottie-swift/src/Private/Model/Animation.swift]: What do you think about renaming this to just `markerNames`?

Comment (dpt) [reply lottie-swift/src/Private/Model/Animation.swift]: Sure. Updated.

File: lottie-swift/src/Private/Model/Animation.swift [hunk 2]
```diff
@@ -57,6 +57,13 @@ public class Animation: Codable {
   let markers: [Marker]?
   let markerMap: [String : Marker]?
   
+  /// Return all marker names, in order, or an empty list if none are specified
+  public var markerNames: [String]
```
Comment (thedrick) [inline lottie-swift/src/Private/Model/Animation.swift]:
```suggestion
  public var markerNames: [String] {
```

File: lottie-swift/src/Private/Model/Animation.swift [hunk 3]
```diff
@@ -57,6 +57,13 @@ public class Animation: Codable {
...
+  {
```
Comment (thedrick) [inline lottie-swift/src/Private/Model/Animation.swift]:
```suggestion
```

File: lottie-swift/src/Private/Model/Animation.swift [hunk 4]
```diff
@@ -57,6 +57,13 @@ public class Animation: Codable {
...
+    guard let markers = markers else { return [] }
+    return markers.map{$0.name}
```
Comment (thedrick) [inline lottie-swift/src/Private/Model/Animation.swift]:
```suggestion
    return markers.map { $0.name }
```

**Comment by dpt** [comment]
Are any other changes needed?

**Pull request merged**